WcaOnRails/app/views/competitions/_registration_requirements.html.erb

```diff
@@ -0,0 +1,64 @@
+<% if @competition.use_wca_registration? %>
+  <%= t('competitions.competition_info.create_wca_account_html', here: link_to(t('common.here'), new_user_registration_path)) %>
+  <br>
+  <%= t('competitions.competition_info.claim_wca_id_html', here: link_to(t('common.here'), profile_claim_wca_id_path)) %>
+  <br>
+  <%= t('competitions.competition_info.register_html', here: link_to(t('common.here'), competition_register_path(@competition))) %>
+  <br>
+<% elsif @competition.website %>
+  <%= t('competitions.competition_info.register_html', here: link_to(t('common.here'), @competition.website, target: "_blank")) %>
+  <br>
+<% else %>
+<% end %>
+<% if @competition.competitor_limit_enabled? %>
+  <%= t('competitions.competition_info.competitor_limit', competitor_limit: @competition.competitor_limit) %>
+  <br>
+<% elsif !@competition.competitor_limit_enabled? %>
+  <%= t('competitions.competition_info.no_competitor_limit') %>
+  <br>
+<% else %>
+<% end %>
+<% if @competition.has_fees? %>
+  <%= t('competitions.competition_info.entry_fee', base_entry_fee: format_money(@competition.base_entry_fee)) %>
+  <br>
+  <% @competition.competition_events.each do |competition_event| %>
+    <% if competition_event.has_fee? %>
+      <dt><%= competition_event.event.name %></dt>
+      <dd><%= format_money competition_event.fee %></dd>
+    <% end %>
+  <% end %>
+<% elsif !@competition.has_fees? %>
+  <%= t('competitions.competition_info.no_entry_fee') %>
+  <br>
+<% else %>
+<% end %>
```

Comment:
Can you add a space between this `end` and the next `if` (and do the same for all the other `if...elif...else...end` blocks in this file? I think having some space between the various attributes will help make this more readable. Right now this file is a pretty intense block of code to look at. I think some whitespace will help a bit.

Reply:
Sure thing!